### PR TITLE
WIP: add thresholds 

### PR DESCRIPTION
**Pull request by wurstbrot**
In order to create the dateperiodComponent for an activity, I need to know for which period which is defined in the `Threshold`.
Therefore, I create the activity multiple times (depending on thresholds number).

**Do not merge until the skeleton.yaml and team definitions are there.**

### CODE REVIEW

File: src/main/java/org/owasp/dsomm/metricca/analyzer/controller/dto/ActivityCount.java
Comment (sumsumcity) [inline src/main/java/org/owasp/dsomm/metricca/analyzer/controller/dto/ActivityCount.java:1]: 1. What is this class needed for? 
2. Should it not have a getter for the entries? 
3. Can we expect activity.getName() to never return null? Maybe a null check

File: src/main/java/org/owasp/dsomm/metricca/analyzer/model/Activity.java
Comment (sumsumcity) [inline src/main/java/org/owasp/dsomm/metricca/analyzer/model/Activity.java:1]: So every activity still has a level? Have you tried that the threshold defines the level?

File: src/main/java/org/owasp/dsomm/metricca/analyzer/model/threshold/Thresholds.java
Comment (sumsumcity) [inline src/main/java/org/owasp/dsomm/metricca/analyzer/model/threshold/Thresholds.java:1]: Getter?

File: test.json
Comment (sumsumcity) [inline test.json:1]: Why do we have the level also in paranthesis

File: src/main/java/org/owasp/dsomm/metricca/analyzer/controller/dto/ActivityCount.java
Comment (wurstbrot) [reply src/main/java/org/owasp/dsomm/metricca/analyzer/controller/dto/ActivityCount.java:1]: the class doesn't exists in the new PR

File: src/main/java/org/owasp/dsomm/metricca/analyzer/model/Activity.java
Comment (wurstbrot) [reply src/main/java/org/owasp/dsomm/metricca/analyzer/model/Activity.java:1]: The class doesn't has a level anymore

File: src/main/java/org/owasp/dsomm/metricca/analyzer/model/threshold/Thresholds.java
Comment (wurstbrot) [reply src/main/java/org/owasp/dsomm/metricca/analyzer/model/threshold/Thresholds.java:1]: has getter in new pr

**Comment by wurstbrot** [comment]
closing this PR due to new one

**Pull request closed without merging**